client-js/app/elements/labrad-nodes.ts

```diff
@@ -46,24 +64,52 @@ export class LabradInstanceController extends polymer.Base {
     this.updateButtonState(newStatus);
   }
 
+
   @listen('start.click')
-  doStart() {
+  async doStart() {
     console.info(`Start: server='${this.name}', node='${this.node}'`);
-    this.api.startServer({node: this.node, server: this.name});
+    try {
+      await this.api.startServer({node: this.node, server: this.name});
+      this.set("server.errorString", "");
+      this.set("server.errorException", "");
+    } catch (e) {
+      console.error("Exception while starting server: ", e);
+      this.set("server.errorString", "An error occurred while starting the server.");
+      this.set("server.errorException", e.message);
+    }
   }
 
+
   @listen('stop.click')
-  doStop() {
+  async doStop() {
     console.info(`Stop: server='${this.name}', node='${this.node}'`);
-    this.api.stopServer({node: this.node, server: this.instanceName});
+    try {
+      await this.api.stopServer({node: this.node, server: this.instanceName});
+      this.set("server.errorString", "");
+      this.set("server.errorException", "");
+    } catch (e) {
+      console.error("Exception while stopping server: ", e);
+      this.set("server.errorString", "An error occurred while starting the server.");
```

Comment:
"...while stopping the server."

Also, as above, please include server name and node name.